code/modules/mob/living/carbon/human/species_types/grod.dm

```diff
@@ -0,0 +1,396 @@
+
+		if("Station Engineer")
+			O = new /datum/outfit/grod/engineer
+
+		if("Atmospheric Technician")
+			O = new /datum/outfit/grod/atmos
+
+		if("Captain")
+			O = new /datum/outfit/grod/captain
+
+		if("Chief Engineer")
+			O = new /datum/outfit/grod/ce
+
+		if("Chief Medical Officer")
+			O = new /datum/outfit/grod/cmo
+
+		if("Head of Security")
+			O = new /datum/outfit/grod/hos
+
+		if("Research Director")
+			O = new /datum/outfit/grod/rd
+
+		if("Head of Personnel")
+			O = new /datum/outfit/grod/hop
+
+		if("Clown")
+			O = new /datum/outfit/grod/clown
+
+		if("Mime")
+			O = new /datum/outfit/grod/mime
+
+		if("Assistant")
+			O = new /datum/outfit/grod/assistant
+	H.equipOutfit(O, visualsOnly)
+	return 0
+
+/datum/species/grod/get_item_offsets_for_index(var/i) //a fall-back incase the mob loses its dir tracking somehow
+	switch(i)
+		if(3) //odd = left hands
+			return list("x" = -1, "y" = 5)
+		if(4) //even = right hands
+			return list("x" = 1, "y" = 5)
+		else
+			return
+
+/datum/species/grod/get_item_offsets_for_dir(var/dir)
+	////BOTTOM LEFT | BOTTOM RIGHT | TOP LEFT | TOP RIGHT
+	switch(dir)
+		if(SOUTH)
+			return list(list("x" = -1, "y" = -4), list("x" = 2, "y" = -4), list("x" = -2, "y" = 1),list("x" = 3, "y" = 1))
+		if(NORTH)
+			return list(list("x" = 3, "y" = 0), list("x" = -2, "y" = 0), list("x" = 3, "y" = 5),list("x" = -2, "y" = 5))
+		if(EAST)
+			return list(list("x" = 4, "y" = -4), list("x" = 10, "y" = -3), list("x" = 6, "y" = 2),list("x" = 10, "y" = 1))
+		if(WEST)
+			return list(list("x" = -10, "y" = -3), list("x" = -4, "y" = -4), list("x" = -10, "y" = 1),list("x" = -6, "y" = 2))
+
+///////Grod Crown////////
+/mob/living/simple_animal/hostile/crown_spider
+	name = "Crownspider"
+	desc = "It looks like a Grod's crown..."
+	icon = 'icons/mob/species/grod/crown_spider.dmi'
+	icon_state = "crown_spider"
+	icon_living = "crown_spider"
+	icon_dead = "crown_spider_dead"
+	gender = NEUTER
+	health = CROWNSPIDER_MAX_HEALTH
+	maxHealth = CROWNSPIDER_BASE_HEALTH
+	melee_damage = 0
+	attacktext = "pinches"
+	attack_sound = 'sound/weapons/bite.ogg'
+	speak_emote = list("squeaks")
+	color = "#00FF00"
+	can_be_held = TRUE
+	worn_slot_flags = ITEM_SLOT_HEAD
+	head_icon = 'icons/mob/species/grod/crown_spider_worn.dmi'
+	held_state = "crown_spider"
+	initial_language_holder = /datum/language_holder/grodcrown //They can only speak Poh'lan in spider
+	var/datum/mind/origin
+
+	pass_flags = PASSMOB
+	speed = 3
+	can_be_held = TRUE
+
+/mob/living/simple_animal/hostile/crown_spider/UnarmedAttack(atom/M)
+	if(try_infect(M))
+		return
+	..()
+
+/mob/living/simple_animal/hostile/crown_spider/MouseDrop(over)
+	. = ..()
+	try_infect(over)
+
+/mob/living/simple_animal/hostile/crown_spider/proc/try_infect(atom/target) //default process before action
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		if(H.stat > CONSCIOUS && (alert("This one is no longer living. Are you sure we should infest it?",,"Yes", "No") == "No"))
```

Comment:
This body makes more sense than This one

Reply:
"This one" is meant to sound off-kilter, because they're aliens.